docs/icicle/colab-instructions.md

```diff
@@ -0,0 +1,131 @@
+# Run Icicle on Google Colab
```

Comment:
@LeonHibnik we should mention  this already setup notebook

https://github.com/gkigiermo/rust-cuda-colab